save-preprocessor/src/main/kotlin/com/saveourtool/save/preprocessor/service/TestDiscoveringService.kt

```diff
@@ -97,17 +106,47 @@ class TestDiscoveringService {
             plugins.asSequence().flatMap { plugin ->
                 plugin.discoverTestFiles(testConfig.directory)
                     .map {
-                        val testRelativePath = it.test.toFile()
-                            .relativeTo(rootTestConfig.directory.toFile())
-                            .path
-                        TestDto(testRelativePath, plugin::class.simpleName!!, testSuite.id!!, it.test.toFile().toHash(), generalConfig.tags!!)
+                        val additionalFiles = if (plugin is FixPlugin) {
+                            listOf((it as FixPlugin.FixTestFiles).expected.getRelativePath(rootTestConfig))
+                        } else {
+                            emptyList()
+                        }
+                        val testRelativePath = it.test.getRelativePath(rootTestConfig)
+                        TestDto(
+                            testRelativePath,
+                            plugin::class.simpleName!!,
+                            testSuite.id!!,
+                            it.test.toFile().toHash(),
+                            generalConfig.tags!!,
+                            additionalFiles,
+                        )
                     }
             }
         }
         .onEach {
             log.debug("Discovered the following test: $it")
         }
 
+    /**
+     * @param pathPrefix path to root test directory
+     * @param testPath path to test file
+     * @return list of strings that are present in the file
+     */
+    fun getTestLinesByPath(pathPrefix: String, testPath: String?) = testPath?.let {
```

Comment:
do we need it as public function?

Reply:
We used to need it but now we don't. Will fix it